vite.config.js

```diff
@@ -46,6 +47,9 @@ function compileMarkdown(content, path) {
 
 export default defineConfig({
 	publicDir: 'src/assets',
+	optimizeDeps: {
+		include: ['@babel/polyfill', '@rollup/browser', 'sucrase']
+	},
```

Comment:
Upon opening `/repl` for the first time in dev, Vite will do a full-page refresh as detects these deps to optimize. Adding them ahead of time avoids that